ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/QuerySubscriber.java

```diff
@@ -106,7 +106,7 @@ public void handleError(final Throwable t) {
     }
     final KsqlErrorMessage errorResponse = new KsqlErrorMessage(
         ERROR_CODE_SERVER_ERROR, stringBuilder.toString());
-    log.error("Error in processing query {}", stringBuilder);
+    log.error("Error in processing query {}", stringBuilder, t);
```

Comment:
We can revert this change. Just reducing the pieces not required to avoid regression.

Reply:
Done